src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
Had some thoughts about this. 

**Concerning the dividers:**
- I would suggest now to not introduce them as own item. There is not much to be gained there. 
- Much more I would introduce a new method on the dropdown itself enabling to add individual sections, e.g. addSection($items). The code to generate with two section would look as follows:
```
$myDropdownWithTwoSections = $f->dropdown()->standard(
     [$entry1section1,$entry2section1]
)->addSection([$entry1section2,$entry2section2]);
```
**Concerning the items:**
- I would still suggest to add links and not dropdown items. To me, those entries are exactly that. I think you would not even need a special renderer for that (since we do not need to change the DOM). Bootstrap/ILIAS can do everything needed through CSS. This also separates dropdowns from Popovers, dropdowns only hold links, Popovers may also contain other elements. One could argue with this separation but it is to some degree an adaption of the bootstrap terminology.
- Concerning your question about an item opening a specific modal, this would look as follows with links:

```
$modal = $factory->modal()->roundtrip('My Modal', $content));
$link = $factory->link()->standard('Open Modal 1', '#')
    ->withOnClick($modal->getShowSignal());
$f->dropdown()->standard( [$link]);
```



- Concerning the question, who will Propose/Implement the links, I could do that.

@alex40724, @klees Major objections to this proposal? If yes we can stick with the dropdown items. In this case I would still suggest to add the addSection Method for the individual sections with dividers.